src/libraries/System.IO.Packaging/tests/Tests.Net.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using Xunit;
+
+namespace System.IO.Packaging.Tests
+{
+    public partial class Tests : FileCleanupTestBase
+    {
+        [Fact]
+        public void Can_Create_Package_On_Unseekable_Stream()
+        {
+            var documentPath = "untitled.txt";
+            Uri partUriDocument = PackUriHelper.CreatePartUri(new Uri(documentPath, UriKind.Relative));
+
+            using (MemoryStream ms = new MemoryStream())
+            using (WrappedStream ws = new WrappedStream(ms, true, true, false))
+            {
+                Package package = Package.Open(ws, FileMode.Create, FileAccess.Write);
```

Comment:
This looks good. It's the one I was expecting to work because we support Create mode with unseekable stream [here](https://source.dot.net/#System.IO.Compression/System/IO/Compression/ZipArchive.cs,160).